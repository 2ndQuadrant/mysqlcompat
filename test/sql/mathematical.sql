\set ECHO none
BEGIN;
\i sql_bits/mathematical.sql
\set ECHO all

SELECT format(10000.01, 0);
SELECT format(10000.01, 2);
SELECT format(0.01, 1);
SELECT format(0, 3);
SELECT format(9.95, 1);
SELECT format(9.94999999, 1);

ROLLBACK;
