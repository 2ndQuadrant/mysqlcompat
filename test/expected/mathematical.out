\set ECHO none
SELECT format(10000.01, 0);
 format 
--------
 10,000
(1 row)

SELECT format(10000.01, 2);
  format   
-----------
 10,000.01
(1 row)

SELECT format(0.01, 1);
 format 
--------
 0.0
(1 row)

SELECT format(0, 3);
 format 
--------
 0.000
(1 row)

SELECT format(9.95, 1);
 format 
--------
 10.0
(1 row)

SELECT format(9.94999999, 1);
 format 
--------
 9.9
(1 row)

ROLLBACK;
